kitsune/announcements/admin.py

```diff
@@ -19,7 +19,7 @@ def is_visible(self, obj):
             return "Yes (always)"
         return ""
 
-    is_visible.short_description = "Visible?"
+    is_visible.short_description = "Visible?"  # type: ignore
```

Comment:
This is required due to an annoying aspect (a flaw, really) of `mypy` where it flags as an error the setting of an attribute on any object if that attribute doesn't already exist.